### PR TITLE
Lecture-7-content-project-7-fix

### PR DESCRIPTION
**Pull request by bcbielecki**

### CODE REVIEW

File: project_7.html [hunk 1]
```diff
@@ -393,22 +388,22 @@ <h3 class="mb-3" id="pj-heading-tm1637">TM1637 7-Segment Display</h3>
 					composed of seven
 					LEDs arranged into segments (as shown in the image below).
 				</p>
-				<img class="img-fluid mb-4" src=".\assets\images\project_7\ops_seven_segment_display.webp">
-				<p class="mb-3">
+				<img class="pj-img-fluid mx-auto mb-4" src=".\assets\images\project_7\ops_seven_segment_display.webp">
+				<p class="mb-5">
```
Comment (dylanvu) [inline project_7.html:392]: I realized that if almost every `<p>` is an `mb-5` maybe we should just write a CSS rule so that we don't have to make each individual bootstrap change?

File: project_7.html [hunk 2]
```diff
@@ -725,19 +736,54 @@ <h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
...
 							<td>1</td>
 						</tr>
 						<tr>
-							<td>TM1637, 7 Segment Display</td>
+							<td>
+								<a class="pj-part"
+									data-bs-content="<img src='assets\images\project_7\ops_TM1637_physical.png' height='150px' width='auto'>"
+									data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0" a
+									class="link-primary"
+									href=""
```
Comment (dylanvu) [inline project_7.html:754]: clicking this opens up this same project page again, is this intended?

File: project_7.html [hunk 3]
```diff
@@ -725,19 +736,54 @@ <h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
...
 							<td>2</td>
 						</tr>
 						<tr>
-							<td>Buzzer, Piezo, 1.5V</td>
+							<td>
+								<a class="pj-part"
+								data-bs-content="<img src='assets/images/parts/ops_part_buzzer.webp' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0" a
+								class="link-primary"
+								href=""
```
Comment (dylanvu) [inline project_7.html:782]: see comment on the TM code

File: assets/images/schematics/ops_schem_proj7_a.png
Comment (dylanvu) [inline assets/images/schematics/ops_schem_proj7_a.png:1]: Was this change intended? The webpage has the circuits flipped compared to the project spec google doc

File: assets/images/schematics/ops_schem_proj7_b.png
Comment (dylanvu) [inline assets/images/schematics/ops_schem_proj7_b.png:1]: Was this change intended? The webpage has the circuits flipped compared to the project spec google doc

**Review by dylanvu — CHANGES_REQUESTED** [review]

File: assets/images/schematics/ops_schem_proj7_b.png
Comment (bcbielecki) [reply assets/images/schematics/ops_schem_proj7_b.png:1]: Not intented. I swapped the photos by accident.

File: project_7.html (hunk 3 above)
Comment (bcbielecki) [reply project_7.html:782]: Resolved. Not intended; excess attributes deleted.

**Review by dylanvu — APPROVED** [review]
